.ci/windows.groovy

```diff
@@ -349,9 +349,10 @@ def mageTargetWin(String context, String directory, String target, String label)
     log(level: 'INFO', text: "context=${context} directory=${directory} target=${target} os=${label}")
     def immutable = label.equals('windows-7-32-bit') ? 'windows-immutable-32-bit' : 'windows-immutable'
 
-    // NOTE: skip filebeat with windows-2016 since there are some test failures.
-    if (directory.equals('filebeat') && label.equals('windows-2016')) {
-      log(level: 'WARN', text: "Skipped stage for the 'filebeat' with 'windows-2016' as long as there are test failures to be analysed.")
+    // NOTE: skip filebeat with windows-2016/2012-r2 since there are some test failures.
+    //       See https://github.com/elastic/beats/issues/19787 https://github.com/elastic/beats/issues/19641
+    if (directory.equals('filebeat') && (label.equals('windows-2016') || label.equals('windows-2012-r2'))) {
+      log(level: 'WARN', text: "Skipped stage for the 'filebeat' with '${label}' as long as there are test failures to be analysed.")
```

Comment:
Could we add the GH issue to the log? So that reading the CI logs will add that information

Reply:
Good point, I'll add it to the other PRs to enable Windows-7, 8, 10 as I need to rebase them too, does it sound good?